app/src/main/java/com/plenart/organizeme/adapters/CardListItemsAdapter.kt

```diff
@@ -6,106 +6,110 @@ import android.view.LayoutInflater
                 val selectedMembersList: ArrayList<SelectedMembers> = ArrayList();
 
-                for(i in context.mAssignedMemberDetailList.indices){
-                    for(j in model.assignedTo){
-                        if(context.mAssignedMemberDetailList[i].id == j){
+                for (i in viewModel.assignedMemberDetailList.value?.indices!!) {
+                    for (j in card.assignedTo) {
+                        if (viewModel.assignedMemberDetailList.value!![i].id == j) {
 
                             val selectedMembers = SelectedMembers(
-                                context.mAssignedMemberDetailList[i].id,
-                                context.mAssignedMemberDetailList[i].image
-                            );
+                                viewModel.assignedMemberDetailList.value!![i].id,
+                                viewModel.assignedMemberDetailList.value!![i].image
+                            )
 
-                            selectedMembersList.add(selectedMembers);
+                            selectedMembersList.add(selectedMembers)
 
                         }
                     }
                 }
 
-                if(selectedMembersList.size > 0){
-                    if(selectedMembersList.size == 1 && selectedMembersList[0].id == model.createdBy){
-                        holder.binding.rvCardSelectedMembersList.visibility = View.GONE;
-                    }
-                    else{
-                        holder.binding.rvCardSelectedMembersList.visibility = View.VISIBLE;
-                        holder.binding.rvCardSelectedMembersList.layoutManager = GridLayoutManager(context,4);
+                if (selectedMembersList.size > 0) {
+                    if (selectedMembersList.size == 1 && selectedMembersList[0].id == card.createdBy) {
+                        binding.rvCardSelectedMembersList.visibility = View.GONE
+                    } else {
+                        binding.rvCardSelectedMembersList.visibility = View.VISIBLE
+                        binding.rvCardSelectedMembersList.layoutManager =
+                            GridLayoutManager(context, 4)
 
-                        val adapter = CardMembersListItemAdapter(context, selectedMembersList,false);
-                        holder.binding.rvCardSelectedMembersList.adapter = adapter;
+                        val adapter =
+                            CardMembersListItemAdapter(context, selectedMembersList, false)
+                        binding.rvCardSelectedMembersList.adapter = adapter
 
-                        adapter.setOnClickListener(object: MemberItemClickInterface{
+                        adapter.setOnClickListener(object : MemberItemClickInterface {
                             override fun onClick() {
-                                if(onClickListener != null){
-                                    onClickListener!!.onClick(position);
+
+                                if (onClickListener != null) {
```

Comment:
`onClickListener?.onClick(position)` -> if onClickListener is null this code will not do anything, if it's not null then onClick function will be invoked